man/journalctl.xml

```diff
@@ -87,18 +87,15 @@
       causes all matches before and after to be combined in a
       disjunction (i.e. logical OR).</para>
 
-      <para>As shortcuts for a few types of field/value matches, file
-      paths may be specified. If a file path refers to an executable
-      file, this is equivalent to an <literal>_EXE=</literal> match
-      for the canonicalized binary path. Similarly, if a path refers
-      to a device node then match is added for the kernel name of the
-      device (<literal>_KERNEL_DEVICE=</literal>). Also, matches for the
-      kernel names of all the parent devices are added automatically.
```

Comment:
Maybe "ancestor" is better than "parent" then.

Nevertheless, I still think your change makes it harder to understand what is going on: before it was obvious that a match for the specified device is added, and going recursively upwards, matches for the parent (ancestor) devices are added. With the revisions it might be understood as a single match for the specified device and it's parents (ancestors). Please keep the sentence "matches for the kernel names of all ... are added automatically".